### PR TITLE
Move routes into its own file and add a basic navbar

### PR DESCRIPTION
**Pull request by dannibaxter**
screen shot 2017-04-16 at 12 46 27 pm

### CODE REVIEW

**Comment by mattjstar** [comment]
Awesome! Can you add some screenshots and a description of what you changed to make it easier for others to immediately look at it?

File: src/common/redux/modules/daycares.js [hunk 1]
```diff
@@ -48,11 +48,7 @@ export const fetchDaycares = () => {
...
       return dispatch({
         type: FETCH_DAYCARE_INDEX_SUCCESS,
-        payload: {
-          entities: {
-            daycares: daycaresState.data,
-          },
-        },
+        payload: daycaresState.data,
       });
     }
```
Comment (dannibaxter) [inline src/common/redux/modules/daycares.js]: This fixes the issue where daycares were only showing up on first load but not after navigating away from the page and coming back

File: src/common/pages/home/home.jsx [hunk 2]
```diff
@@ -7,8 +8,10 @@ class Home extends Component {
       <div className={style.container}>
         <h1 className={style.title}>Karl and Keke</h1>
         <h2 className={style.subtitle}>Daycare Search and Ratings in New York City</h2>
+        <input className="search-input" placeholder="Search by daycare or zipcode">{}</input>
```
Comment (mattjstar) [inline src/common/pages/home/home.jsx]: shorthand: `<input className="search-input" placeholder="Search by daycare or zipcode" />`

File: src/common/redux/modules/daycares.js (hunk 1 above)
Comment (mattjstar) [inline src/common/redux/modules/daycares.js]: Huh, why did getting rid of entities make that work?

Comment (dannibaxter) [inline src/common/redux/modules/daycares.js]: it got rid of this warning (I think) 
`warning.js?8a56:36 Warning: Failed prop type: Invalid prop `daycares` of type `object` supplied to `Search`, expected an array.
    in Search (created by Connect(Search))
    in Connect(Search) (created by Route)
    in Route (created by Routes)
    in div (created by Routes)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by Routes)
    in Routes
    in Provider`
and stopped the array from getting nested within another object on second load

**Pull request merged**